Source/Plugins/EditorModules/SceneView/SceneViewSettings.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Duality.Editor.Plugins.SceneView
+{
+	public class SceneViewSettings
+	{
+		private bool showComponents = true;
+
+		public bool ShowComponents
```

Comment:
Class and property need an XML comment. (The property controls whether components are shown as individual nodes in the scene tree.)